src/libs/ValidationUtils.js

```diff
@@ -206,7 +206,7 @@ function isValidURL(url) {
  * @returns {Object}
  */
 function validateIdentity(identity) {
-    const requiredFields = ['firstName', 'lastName', 'street', 'city', 'zipCode', 'state', 'ssnLast4', 'dob'];
+    const requiredFields = ['firstName', 'lastName', 'addressStreet', 'addressCity', 'addressZipCode', 'addressState', 'ssnLast4', 'dob'];
```

Comment:
This change is because I changed these names in the IdentityForm.js for consistency, see lines:

https://github.com/Expensify/App/blob/970c2d528e401956fc9702f31b48fe226630a5be/src/pages/ReimbursementAccount/IdentityForm.js#L155-L156

IdentityForm is reading values using one key (i.e. `state`) and then calling `onChangeText` with a different key (i.e. `addressState`). This just makes it harder to follow and update the logic where we already have some key renaming (i.e. requestor step, beneficial owners).

Reply:
Well, looking at it again, this function could really stay as it is. The thing that was confusing is the IdentityForm reading an input with one key and then calling "onFieldChange" with another key.